.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+name: release
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: choice
+        options:
+          - sbx
+          - prod
+          - prod-test  
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  # Promote and Deploy to prod-test
```

Comment:
Add a note to comment that prod-test only includes worker, and drop the promote-prod-test-api job.